tests/testthat/test-geom-hline-vline-abline.R

```diff
@@ -43,3 +43,38 @@ test_that("curved lines in map projections", {
     nzmap + coord_map(projection = 'azequalarea', orientation = c(-36.92, 174.6, 0))
   )
 })
+
+# Warning tests ------------------------------------------------------------
+
+test_that("Warning if a supplied mapping is going to be overwritten",{
+  dat <- data.frame(x = LETTERS[1:5], y = 1:5,colour = c(TRUE,TRUE,rep(FALSE,3)))
```

Comment:
There's no need to construct a plot here since the warning comes from the `geom_vline()` call.